src/Control/Retry.hs

```diff
@@ -31,7 +32,19 @@ module Control.Retry
       RetryPolicyM (..)
     , RetryPolicy
     , retryPolicy
-
+    , RetryStatus
+    -- ** Fields for 'RetryStatus'
+    , rsIterNumber
+    , rsCumulativeDelay
+    , rsPreviousDelay
+    , defaultRetryStatus
+
+    -- ** Lenses for 'RetryStatus'
+    , rsIterNumberL
+    , rsCumulativeDelayL
+    , rsPreviousDelayL
```

Comment:
Maybe these lenses shouldn't be exported for now because they haven't been implemented (yet). Or are you going to implement it before merging?